presto-orc/pom.xml

```diff
@@ -211,5 +211,139 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>java21PrestoORC</id>
+            <activation>
+                <jdk>21</jdk>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.mockito</groupId>
```

Comment:
We don't use mocks. Please create stubbed out classes instead.

Reply:
Could you kindly direct me to an example within Presto? That would be very helpful.

Reply:
you can look at any classes named TestingXXX as an example. We create testing implementations of interfaces that follow the needs of a test. Can be simple or complex as needed.   Here are a few examples, and you can see how they are used in tests:  https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/testing/TestingMetadata.java, https://github.com/prestodb/presto/blob/master/presto-spi/src/test/java/com/facebook/presto/spi/ttl/TestingNodeTtlFetcher.java, https://github.com/prestodb/presto/blob/master/presto-spi/src/test/java/com/facebook/presto/spi/TestingColumnHandle.java, https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/testing/TestingAccessControlManager.java